Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -354,6 +541,35 @@ bool isFunctionJson(string fullName)
             }
         }
 
+        private async Task LocalAppFetch(IRepository repository, DeploymentInfoBase deploymentInfo, string targetBranch, ILogger logger, ITracer tracer)
+        {
+            var zipDeploymentInfo = (ZipDeploymentInfo)deploymentInfo;
+            var content = !string.IsNullOrEmpty(zipDeploymentInfo.ZipURL)
```

Comment:
>ZipURL [](start = 66, length = 6)

Will the URL point to a zip if type=war (app.war) ?

Reply:
No, I began coding this with the mindset of trying to change as little as possible of the existing code. I have recently realized that that mindset has made my code kind of difficult to read.